routing/pathfind_test.go

```diff
@@ -310,8 +490,13 @@ func TestBasicGraphPathFinding(t *testing.T) {
 	)
 
 	paymentAmt := lnwire.NewMSatFromSatoshis(100)
-	target := aliases["sophon"]
-	path, err := findPath(nil, graph, sourceNode, target, ignoredVertexes,
+	target := NewVertex(aliases["sophon"])
+
+	g, err := newMemChannelGraphFromDatabase(graph)
```

Comment:
A more complete test here would be to modify all the graph tests to run using both the DB and in-memory versions.

Reply:
> using both the DB and in-memory versions.

Since the DB functions don't exactly match graphSource functions, I've made a wrapper for it `dbChannelGraph`, I think it's `ForEachChannel` should be improved(as mentioned elsewhere).